includes/import-products.php

```diff
@@ -90,22 +105,31 @@ function dw_add_woocommerce_product($course) {
 			'value'       => $startDate->format('H:i'),
 			'is_visible'  => true,
 			'is_taxonomy' => '1'
-		]
+		],
+		'pa_slot_time' => [
+			'name'     => 'pa_slot_time',
+			'is_visible' => true,
+			'is_taxonomy' => true,
```

Comment:
All other `is_taxonomy` keys have value `'1'`. Is there a reason for that?

Reply:
Nope, since both ʻ1ʻ and true seem to work, Iʻll change all of ʻm to true